docs/troubleshooting.md2

```diff
@@ -0,0 +1,91 @@
+# What
+
+This document serves as the knowledge base for troubleshooting the Open Data Hub Operator.
+More information can be found at https://github.com/opendatahub-io/opendatahub-operator/wiki
+
+## Troubleshooting
+
+### Upgrade from Operator v2.0/v2.1 to v2.2+
+
+This also applies to any local build deployment from the "main" branch.
+
+To upgrade, follow these steps:
+
+- Disable the component(s) in your DSC instance.
```

Comment:
Do we need to deactivate the components, or does deleting the DSC do the same thing?

Reply:
this is an old upgrade case since we have changes in 2.0/2.1 to 2.4 GA version.
most of the current user should not encounter this, unless they have been on 2.0/2.1 for 1.5yrs?
maybe we can revisit these documents and clean up some later

Reply:
Sounds good, we can revisit and clean them up later.